hardware_interface/CMakeLists.txt

```
@@ -40,9 +40,23 @@ target_compile_features(hardware_interface PUBLIC cxx_std_17)
 target_include_directories(hardware_interface PUBLIC
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/hardware_interface>
+  ${pal_statistics_INCLUDE_DIRS}
 )
-target_link_libraries(hardware_interface PUBLIC fmt::fmt)
-ament_target_dependencies(hardware_interface PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
+target_link_libraries(hardware_interface PUBLIC
+                      rclcpp::rclcpp
+                      rclcpp_lifecycle::rclcpp_lifecycle
+                      pluginlib::pluginlib
+                      urdf::urdf
+                      realtime_tools::realtime_tools
+                      rcutils::rcutils
+                      rcpputils::rcpputils
+                      ${joint_limits_TARGETS}
+                      ${TinyXML2_LIBRARIES}
+                      ${tinyxml2_vendor_LIBRARIES}
+                      ${pal_statistics_LIBRARIES}
```

same like https://github.com/ros-controls/realtime_tools/pull/331#discussion_r2101051854, is there another way instead of these `_LIBRARIES`?

---

```suggestion
                      tinyxml2::tinyxml2
                      ${pal_statistics_LIBRARIES}
```

For pal_statistics, this works for now, we can fix it in future

---

Checking ament_auto_cmake, it exports targets as `_LIBRARIES`
https://github.com/ament/ament_cmake/blob/humble/ament_cmake_auto/cmake/ament_auto_package.cmake#L91-L99